contracts/payment/PaymentSplitter.sol

```diff
@@ -22,7 +22,11 @@ contract PaymentSplitter {
     address[] private _payees;
 
     /**
-     * @dev Constructor.
+     * @dev Creates an instance of `PaymentSplitter` where each account in `payees` is assigned the number of shares at
+     * the matching position in the `shares` array. Each account will then receive payments proportional to the
+     * percentage of the shares that they were assigned.
+     *
+     * Both arrays must have the same non-zero length, and there must be no duplicates in `payees`.
```

Comment:
Payees must also not be the zero address, and shares must be non-zero, but this requirement is in `_addPayee` (a private function). What do?

Reply:
The requirement propagates to the constructor. I don't see any problem here. Added it.